packages/@react-aria/tooltip/docs/useTooltipTrigger.mdx

```diff
@@ -0,0 +1,132 @@
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/tooltip';
+import statelyDocs from 'docs:@react-stately/tooltip';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-aria/switch/package.json';
+
+```jsx import
+import {useTooltipTrigger} from '@react-aria/tooltip';
+import {useTooltip} from '@react-aria/tooltip';
+```
+
+---
+category: Overlays
+keywords: [tooltip, aria]
+---
+
+# useTooltipTrigger
+
+<p>{docs.exports.useTooltipTrigger.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useTooltipTrigger']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/TR/wai-aria-1.2/#tooltip'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useTooltipTrigger} links={docs.links} />
+
+## Features
+
+There is no native HTML element for tooltips. There are many guides out there for
+tooltips, however, they are not easy to make accessible, especially the CSS only ones.
+
+* Keyboard focus management and cross browser normalization
+* Hover management and cross browser normalization
+* Labeling support for screen readers (aria-describedby)
+* Exposed as a tooltip to assistive technology via ARIA
+
+## Anatomy
+
+A tooltip consists of two parts, the target trigger and the tooltip itself.
+Users may reveal the tooltip by hovering or focusing the target trigger.
+
+`useTooltipTrigger` returns props to be spread onto its target trigger and the tooltip:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTooltipTrigger.return.id].properties} />
+</TypeContext.Provider>
+
+Tooltip state is managed by the <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTooltipTriggerState} />
+hook in `@react-stately/tooltip`. The state object should be passed as an option to `useTooltipTrigger`.
+
+## Example
+
+This example uses a Tooltip that renders in an absolute position next to its target. We need to supply it with the correct
+aria attributes, which we can get from the <TypeLink links={docs.links} type={docs.exports.useTooltip} /> hook also in `@react-aria/tooltip`.
+
+```tsx example
+import {useTooltipTriggerState} from '@react-stately/tooltip';
+import {
+  useOverlay,
+  useOverlayPosition,
+} from '@react-aria/overlays';
+import {mergeProps} from '@react-aria/utils';
+
+function Tooltip(props) {
+  let {tooltipProps} = useTooltip(props);
+
+  return (
+    <span
+      style={{
+        position: 'absolute',
+        right: '-10px',
+        top: '0',
+        transform: 'translateX(100%)',
+        backgroundColor: 'white',
+        color: 'black',
+        padding: '5px'
+      }}
+      {...mergeProps(props, tooltipProps)} >
+      {props.children}
+    </span>
+  );
+}
+
+function TooltipedButton(props) {
```

Comment:
kinda strange name :D maybe just `Example` or `TooltipExample` since it's not really a component